src/domain/account/account.repository.interface.ts

```diff
@@ -13,14 +13,14 @@ export interface IEmailRepository {
   }): Promise<string[]>;
 
   /**
-   * Saves an email entry.
+   * Saves an account entry.
    *
    * @param args.chainId - the chain id of where the Safe is deployed
    * @param args.safeAddress - the Safe address to which we should store the email address
    * @param args.emailAddress - the email address to store
    * @param args.account - the owner address to which we should link the email address to
```

Comment:
If we agree on renaming this to `createAccount` then we should update this.

Reply:
[fa63a4e](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/fa63a4e9950036c1b044c936774359b805c7c62f)